spec/system/sign_in_spec.rb

```diff
@@ -17,8 +17,11 @@
     it "takes user to home page" do
       visit "/"
       click_button "Start now"
+      expect(page).to have_selector(".govuk-header__navigation-item--active")
```

Comment:
i don't think we need this line as you are checking the correct link is "active" below, which is a preferred way

Reply:
Thanks for the tip. I have deleted this line now.